src/browser/navigation-panel.js

```diff
@@ -44,8 +44,8 @@ define((require, exports, module) => {
   const inputBindings = KeyBindings({'escape': focus});
 
 
-  const NavigationControls = Component('NavigationControls', ({inputCursor, tabStripCursor,
-                                         webViewerCursor, theme}) =>
+  const NavigationControls = Component('NavigationControls',
+    ({inputCursor, tabStripCursor, dashboardCursor, webViewerCursor, theme}) =>
```

Comment:
Suggestion: We should not pass cursors unless they affect render results, instead we should pass handler functions to trigger updating those cursors, as updating event handlers won't re-trigger components while updates to state under cursor will. At this point I suggest to either submit an issue to do it later on and put TODO: here so it's easy to find places that will require updates.

We would need to update to new version of omniscient to get swapping of event handlers without re-renders that I've contributed.